src/der.rs

```diff
@@ -14,6 +14,8 @@
 
 use core::marker::PhantomData;
 
+use alloc::vec::Vec;
```

Comment:
I think you'll need to gate this import on `feature = "alloc"`:

```
17 | use alloc::vec::Vec;
   |     ^^^^^ use of undeclared crate or module `alloc`
```

Reply:
Gated this in [ef13aaa](https://github.com/rustls/webpki/pull/253/commits/ef13aaa72d8377055b4b56406ab63ac774e2b358): 

```diff
+ #[cfg(feature = "alloc")]
use alloc::vec::Vec;
```